### PR TITLE
Add FIWARE Processing Chapter CSS and doc badges

### PR DESCRIPTION
**Pull request by jason-fox**
Adds Standard Badges and CSS as agreed by the Technical Steering Committee.

* Deleted Custom CSS
* Standardized License file name (removed suffix)

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -28,12 +28,4 @@ pages:
         - 'Testing': 'development/platform/testing.md'
         - 'Translations': 'development/platform/translation.md'
 
-markdown_extensions:
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply mkdocs.yml]: My mistake - I deleted them whilst testing locally.

[the remark this answers is not in the document]
Comment (jason-fox) [reply mkdocs.yml]: Re-added `markdown_extensions` - Fixed - d9df57109

File: README.md [hunk 2]
```diff
@@ -42,10 +43,16 @@ You can find the User & Programmer's Manual and the Install & Administration Gui
...
 
 See the [installation guide](https://wirecloud.readthedocs.io/en/stable/installation_guide/) for more info.
 
 ## Bugs and feature requests
 
 Have a bug or a feature request? Please first read the [issue guidelines](CONTRIBUTING.md#using-the-issue-tracker) and search for existing and closed issues. If your problem or idea is not addressed yet, [please open a new issue](https://github.com/Wirecloud/wirecloud/issues/new).
+
+## License
+
+Wirecloud is licensed under Affero General Public License (GPL) version 3 with classpath-like exception
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply README.md]: Fixed - 6cfb183f - Text amended

File: README.md [hunk 3]
```diff
@@ -7,10 +7,11 @@
...
+[![License: AGPL](https://img.shields.io/github/license/wirecloud/wirecloud.svg)](https://opensource.org/licenses/AGPL-3.0)
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply README.md]: Fixed 3224a9a9 - I get your point and I think I agree - it only makes sense to  link to a "Standard Boilerplate" APGL if the software is pure AGPL - I've altered link to local license. 

**However**,  the image for the badge is the standard AGPL one provided by `shields.io` - this is all part of attempt to standardize look-and-feel. This aligns with the current FIWARE developerGuideline.

Personally I don't think this is an issue -  the text of the **badge**  is merely a summary of the license type. To clarify it in more detail, a reader can read the text at the base of the `README` (which is the defacto standard place to find it) - or indeed read the `LICENSE` file itself (which can already be found in the standard location.

`shields.io` is effectively proving that the `LICENSE` file exists and has been identified as an AGPL type.

**Comment by aarranz** [comment]
Thank you for your contribution Jason 😃.

**Pull request merged**